src/PKSim.Presentation/Presenters/Simulations/SimulationWizardPresenter.cs

```diff
@@ -73,7 +73,7 @@ protected override void UpdateControls(int indexThatWillHaveFocus)
 
       public virtual void CreateSimulation()
       {
-         var success = _heavyWorkManager.Start(UpdateSimulationProperties, HeayWorkCaption);
+         var success = _heavyWorkManager.Start(UpdateSimulationProperties, HeavyWorkCaption);
```

Comment:
I always correct typo now that they are shown clearly with the plugin I am using (maybe it's part of resharper?)